src/widgets/ConfigWindow.cpp

```diff
@@ -83,11 +84,7 @@ ConfigWindow::ConfigWindow(QWidget *parent)
 
 	QGroupBox *textEditor = new QGroupBox(tr("Text Editor"), this);
 
-	windowColor1 = new QPushButton(textEditor);
-	windowColor2 = new QPushButton(textEditor);
-	windowColor3 = new QPushButton(textEditor);
-	windowColor4 = new QPushButton(textEditor);
-	windowPreview = new QLabel(textEditor);
+    windowPreview = new DialogPreview(textEditor);
```

Comment:
please use tabs with spaces for continuation

Reply:
Done